packages/customize-widgets/src/components/customize-widgets/use-clear-selected-block.js

```diff
@@ -52,20 +52,23 @@ export default function useClearSelectedBlock( sidebarControl, popoverRef ) {
 				}
 			}
 
-			// Handle focusing in the same document.
-			function handleFocus( event ) {
+			// Handle mouse down in the same document.
+			function handleMouseDown( event ) {
 				handleClearSelectedBlock( event.target );
 			}
 			// Handle focusing outside the current document, like to iframes.
 			function handleBlur() {
 				handleClearSelectedBlock( ownerDocument.activeElement );
 			}
 
-			ownerDocument.addEventListener( 'focusin', handleFocus );
+			ownerDocument.addEventListener( 'mousedown', handleMouseDown );
```

Comment:
@kevin940726 I switched to mousedown to copy the `useBlockSelectionClearer` hook. Are there any cases you know of where this won't be adequate?

Reply:
Not that I know of 🤔. I just thought that `focus` is a more appropriate event 😅. Does `mousedown` work in mobile too?

Reply:
Seems to be fine on mobile. 👍